cmd/execution/main.go

```diff
@@ -308,10 +308,13 @@ func main() {
 		Component("ingestion engine", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
 			collectionRequester, err = requester.New(node.Logger, node.Metrics.Engine, node.Network, node.Me, node.State,
 				engine.RequestCollections,
-				filter.HasRole(flow.RoleCollection),
+				filter.Any,
```

Comment:
I assume we do collection hash somewhere that we let this open to any

Reply:
Yep - https://github.com/onflow/flow-go/blob/36069d4273be79fe41268c6abac8a4eb150412a0/engine/common/requester/engine.go#L447